Microsoft.DotNet.Interactive.PowerShell/Host/PSKernelHostRawUserInterface.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Management.Automation.Host;
+
+namespace Microsoft.DotNet.Interactive.PowerShell.Host
+{
+    public class PSKernelHostRawUserInterface : PSHostRawUserInterface
+    {
+        private Size _bufferSize;
+        private ConsoleColor _foregroundColor, _backgroundColor;
+
+        internal PSKernelHostRawUserInterface()
+        {
+            _bufferSize = new Size(100, 50);
```

Comment:
What's the reason for these specific dimensions?

Reply:
No specific reason. When there is no host implementation, the default width used by the PowerShell formatting system is 120, and it assumes infinite console height. I tried 120, and it's a little too much on my laptop, so I choose 100 as the default. User can configure this by setting `$host.UI.RawUI. WindowSize`.

Reply:
BTW, the width and height are needed for the powershell formatting to properly generate the texts that representing the result objects.